.travis.yml

```diff
@@ -52,24 +42,21 @@ before_install:
 
 
   - scan_test() {
-        if [ $id == "devel-1" ]; then
-          run_test mysql_create.yml;
-          run_test postgresql_create.yml;
-          run_test sql_create.yml;
-        fi;
-        if [ $id == "devel-2" ]; then
-          run_test aci_create_with_acr.yml;
-        fi;
-        if [ $id == "devel-3" ]; then
-          run_test vm_create.yml;
-          run_test vm_create_ssh.yml;
-        fi;
-        if [ $id == "devel-4" ]; then
-          run_test networkinterface_with_virtualnetwork_in_another_resource_group.yml;
-        fi;
-        if [ $id == "devel-5" ]; then
-          echo "no tests";
-        fi;
+      end=$((id * number_per_job));
+      start=$((end - number_per_job));
+
+      liststr=./*.yml;
+      list=($liststr);
+
+      echo start = $start, end = $end, list_lenth = ${#list[@]};
+
+      for (( i = $start; i < $end; i++ )); do	
+        if [ "$i" -lt "${#list[@]}" ]; then
+          if [ "${list[$i]}" != "./vm_create_existingvnet_deployjavaapp.yml" ]; then
```

Comment:
ignore `vm_create_existingvnet_deployjavaapp.yml`? Can we make this a config list? Since Cao Hai is adding AKS sample, which should be excluded in the CI

Reply:
agree, will make a excluded list.

Reply:
i think we should still have include list rather than exclude list.
if we had include list, implemented as i did before, it would be pretty easy to add tests even from subdirectories, etc.
some tests may have helper yaml files, etc.
some other tests may need to run playbooks in a particular order.
if we have include list, rather than scan for yaml files, it could be all handled well.

Reply:
for testing subfolder, i'll send in next pr.